css/style.css

```diff
@@ -1,211 +1,448 @@
-body {
+/* ----------Fonts---------- */
+
+.title,
+.h1Top,
+.h1Bottom,
+.linkHeader,
+.sendButton,
+.sectionButton,
+.subscribeButton {
 	font-family: Georgia, 'Times New Roman', Times, serif;
 }
+
+.linkDescription {
+	font-family: Tahoma, 'Segoe UI', Geneva, Verdana, sans-serif;
+}
+
+footer,
+.underImg,
+.section p,
+.buttonDiv,
+.underTitle,
+.bottomRightIcons p {
+	font-family: Helvetica, Arial, sans-serif;
+}
+
+.h1BottomBottom {
+	font-family: 'Gill Sans MT', 'Gill Sans', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+.namePerson,
+.messagePerson,
+.emailPersonLeft,
+.emailPersonCenter {
+	font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
```

Comment:
What is the reason of useing several font-families? (es commentin petq a takic patasxanes)

Reply:
The photo of the Polo 360 template uses different fonts. I wanted to make it as close to the original as possible.